File: common-auth/src/main/java/com/reservation/commonauth/auth/config/JwtProperties.java
package com.reservation.commonauth.auth.config;

import java.util.List;

import org.springframework.boot.context.properties.ConfigurationProperties;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
@ConfigurationProperties(prefix = "jwt")
public class JwtProperties {
	private String secretKey;
	private long accessTokenExpiry;
	private long refreshTokenExpiry;
	private List<String> skipUrls;
}

File: gateway-service/src/main/kotlin/com/msa/gatewayservice/config/properties/JwtProperties.kt
package com.msa.gatewayservice.config.properties

import jakarta.validation.constraints.NotBlank
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.validation.annotation.Validated


@Validated
@ConfigurationProperties(prefix = "jwt")
data class JwtProperties(
    @field:NotBlank(message = "jwt 시크릿 키가 비어있을 수 없습니다.")
    val secretKey: String,
)
